workflow/exp/exp.ens_conus12km

```diff
@@ -74,3 +74,43 @@ export WALLTIME_JEDIVAR="00:30:00"
 #
 export KEEPDATA=yes
 export MPI_RUN_CMD=srun
+#
+# local.setup: setup some local options specific to a platform or an experiment
```

Comment:
With all these settings, do you still need to have the above settings such as `export PARTITION="local.setup"`?

Reply:
Yes, all "local.setup" variables may be removed in the future.
I keep them right now to facilitate the comparison between the realtime versions and retro versions.